nav2_bringup/bringup/launch/nav2_namespaced_rviz_launch.py

```diff
@@ -0,0 +1,85 @@
+# Copyright (c) 2018 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+from ament_index_python.packages import get_package_share_directory
+
+from launch import LaunchDescription
+from launch.actions import DeclareLaunchArgument, EmitEvent, RegisterEventHandler
+from launch.conditions import IfCondition
+from launch.event_handlers import OnProcessExit
+from launch.events import Shutdown
+from launch.substitutions import LaunchConfiguration
+
+from nav2_common.launch import Node, ReplaceString
+
+
+def generate_launch_description():
```

Comment:
Is there a reason the normal (ei your other `nav2_rviz_launch.py`) RVIZ launch file can't be the "namespaced" config where the default namespace is just no namespace at all?

Our launch file directory is starting to get a little redundant. I agree with you separating out the rviz launch file so that we can launch it separately / have it optional so we have a consistent rviz window between multiple simulation sessions, but the few additions I see here could just be logical namespace-less defaults and remove the second file.

Reply:
Agree, just pushed changes to address this.